tmt/steps/provision/podman.py

```diff
@@ -44,6 +46,26 @@ class PodmanGuestData(tmt.steps.provision.GuestData):
         metavar='NAME',
         help='Name or id of an existing container to be used.')
 
+    pull_attempts: int = field(
+        default=DEFAULT_PULL_ATTEMPTS,
+        option='--pull-attempts',
+        metavar='ATTEMPT',
```

Comment:
```suggestion
        metavar='COUNT',
```

Could a bit more clear?